assets/stylesheets/bootstrap/mixins/_buttons.scss

```diff
@@ -72,24 +72,22 @@
   }
 }
 
-@mixin button-variant($color, $background, $border) {
```

Comment:
`$border` is always the same as `$background` for this button, so do as we did for `button-variant-plain` above and remove the extra param